src/main/resources/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig/help-deleteBlobs.html

```diff
@@ -0,0 +1 @@
+<div>this parameter is set by the property <code>-Dio.jenkins.plugins.artifact_manager_s3.S3BlobStoreConfig.deleteBlobs</code> enable deleting blobs when a build is deleted</div>
```

Comment:
I find it confusing, is it is set by the property or can it be edited in the config?
If it is the latter I would just say something like 

If not checked the blobs will not be deleted from S3 when the corresponding build is deleted.

Reply:
it is set with a property now, maybe we can move it to be configurable, I did not because I do not want to add more changes, but it is a simple one.

Reply:
I think it is OK as is—there is a view-only checkbox whose help describes how the setting is configured.

(Now whether we should switch to a GUI-configurable checkbox defaulting to no deletion, and drop the system property, is another question.)